src/libstd/keyword_docs.rs

```diff
@@ -710,3 +789,79 @@ mod loop_keyword { }
 /// [book]: ../book/ch05-01-defining-structs.html
 /// [reference]: ../reference/items/structs.html
 mod struct_keyword { }
+
+#[doc(keyword = "super")]
+//
+/// The parent of the current [module].
+///
+/// [module]: ../../nightly/book/ch07-02-defining-modules-to-control-scope-and-privacy.html
+mod super_keyword { }
+
+#[doc(keyword = "trait")]
+//
+/// A single interface shared across types.
+mod trait_keyword { }
+
+#[doc(keyword = "true")]
+//
+/// A value of type `bool` representing logical **true**.
+mod true_keyword { }
+
+#[doc(keyword = "type")]
+//
+/// Give a new name to an existing type.
```

Comment:
Not really true for associated types.

Reply:
There are four uses that I'm aware of: `type` aliases, declaring an associated type in a trait definition, defining an associated type in a trait implementation, and `extern type`. I think that this statement also applies when defining an associated type (I give a type the name `<T as Trait>::AssocType`) and declaring an `extern type` (I give some external type a name, but don't specify its layout). It doesn't really apply when defining a trait with an associated type. 

Perhaps this is good enough?

Reply:
Yeah fair enough, but it would be good to elaborate more later.

At your option, here's one possible change:
```suggestion
/// Define an alias for an existing type.
```